mne_bids/read.py

```diff
@@ -344,6 +356,11 @@ def read_raw_bids(bids_fname, bids_root, extra_params=None,
     extra_params : None | dict
         Extra parameters to be passed to MNE read_raw_* functions.
         If a dict, for example: ``extra_params=dict(allow_maxshield=True)``.
+    space : None | str
```

Comment:
why do you need this argument? You should get the `space` from `coordsystem.json`, no?

Reply:
If <MEG/EEG/IEEG>CoordinateSystem key in the coordsystem json is one of the keywords allowed per modality, then not needed. (I am pretty sure?) 

However, if it is "Other", then the `space` entity is needed in order to discern what the coordinate system is. E.g. MNI, etc. This is the only way I could see that would still follow the BIDS specification, and allowing some flexibility in not using their recommended coordinate system formats (imo too stringent rn since there's no easy way to automate ACPC reorientation).

Reply:
Adding more arguments to this function can be a bad precedent. We should try to read as much possible and throw warnings to encourage users to have a more complete dataset that can be read into MNE. This just corresponds to filling up one entry in one file right?

Reply:
I agree with @jasmainak 

don't try to do too many things in one PR @adam2392 

this one was pretty close to merge and we need to rediscuss again and maybe revert/adjust

Reply:
Okay that's a fair point. I had included the space parameter to get around having multiple electrodes.tsv files, but if we want to proceed w/ only allowing one per dataset when reading, then that is fine, else warn(?). 

Let me know what you think and I can change accordingly.

Reply:
yes that sounds fair to me. BIDS has too many bells and whistles, let's make it work in stages incrementally.